packages/material-ui-codemod/package.json

```diff
@@ -26,12 +27,15 @@
   "homepage": "https://github.com/mui-org/material-ui/tree/next/packages/material-ui-codemod",
   "dependencies": {
     "@babel/core": "^7.4.5",
+    "@babel/runtime": "^7.4.4",
     "@babel/traverse": "^7.6.2",
-    "jscodeshift-add-imports": "^1.0.1"
+    "lodash": "^4.17.15",
```

Comment:
Can be removed once https://github.com/facebook/jscodeshift/pull/432 is released